VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionComment.java

```diff
@@ -41,6 +48,8 @@
     private boolean abuseFlagged = false;
     private List<String> editableFields;
     private int childCount = 0;
+    @Nullable
```

Comment:
I suppose it may be out of scope here, but all the fields in these models should be declared as either `@Nullable` or `@NonNull`.